tests/features/publication.feature

```diff
@@ -4,29 +4,95 @@ Feature: Publication content creation
   As an editor
   I need to be able to create and see publication items
 
+  @javascript
   Scenario: Creation of a Publication content through the UI.
-    Given I am logged in as a user with the "create oe_publication content, access content, edit own oe_publication content, view published skos concept entities" permission
-    # Create a "Document".
+    Given I am logged in as a user with the "create oe_publication content, access content, edit own oe_publication content, view published skos concept entities, manage corporate content entities" permission
+
     And the following documents:
     | name          | file       |
     | My Document 1 | sample.pdf |
-    # Create a "Publication" content.
+
+    And the following images:
+    | name         | file           | alt          |
+    | Sample image | example_1.jpeg | example text |
+
+    And the following General Contact entity:
+    | Name               | A general contact                                                                         |
+    | Address            | country_code: BG - locality: Varna - address_line1: General contact 1 - postal_code: 9009 |
+    | Email              | general@example.com                                                                       |
+    | Phone number       | +359525566778                                                                             |
+    | Social media links | uri: http://instagram.com - title: Instagram - link_type: instagram                       |
```

Comment:
Where do you use this? On line 48 you start creating a new contact.

Reply:
I removed this entity definition.